deployments/charts/kuma/templates/cp-deployment.yaml

```diff
@@ -165,9 +165,23 @@ spec:
               cpu: 100m
               memory: 256Mi
             {{- end }}
-            {{- if .Values.controlPlane.resources.limits }}
             limits:
+            {{- if .Values.controlPlane.resources.limits }}
             {{ .Values.controlPlane.resources.limits | toYaml | nindent 14 }}
+            {{- else if .Values.controlPlane.resources.requests }}
+            {{- /* 
+              This "double if" is necessary to satisfy the version of `helm lint` currently in use, v3.8.2.
+              With `if and`, it complains about `requests` being `nil` and the `requests.memory` check
+              because it doesn't understand the `requests` check eliminates `requests` being nil.
+              It can be collapsed into the outer "if" with Helm v3.11.1 at the latest.
+            */}}
```

Comment:
Looks like we should do something so that our helm install and go dependency version stay in sync: https://github.com/kumahq/kuma/blob/9ed757d3e5955ea57d7940badf4468057ed46663/mk/dependencies/helm.sh#L6